### ISSUE TITLE
v2014.08-1 and v2014.11-1 are crashing on Windows 8.1

### ISSUE THREAD

**Issue by mvdklip** [speps/XInputDotNet#18]
I have been using older versions of this project with success, but after upgrading to newer versions recently my project has started crashing. More specifically if I try to run either the 08-1 or the 11-1 versions on a Win8.1 64-bit system. If I try to run the same build on a Win7 64-bit system it works fine.

Today I cloned the project and compiled the project myself which seems to fix the issues, so this might already been addressed. In that case, when are you planning to do a release?

**Comment by speps** [comment]
I need to make builds with the fixes @hymerman did. You can try building it locally using the .bat files.

**Comment by mvdklip** [comment]
As reported earlier, building locally seems to fix the issues so that's why I would gladly see an official release. FYI XInputDemo.exe from the mentioned releases is also crashing on this Win8.1 64-bit system.

**Comment by spiked3** [comment]
I think I am running into related problems. Win 8.1 64bit but with VS 2015 RC.  My apps are acting really strange. One is running fine at the moment, the other refuses to run at all, invalid file format loading .dll.  Both projects created identically. I have spent 2 days trying to resolve the problem and have had no luck. In the past even the app that is running at the moment will start getting the same error eventually.  Whenever I try to load the provided solution, it crashes vs2015. I have loaded it in VS 2013 and it runs (except crashes at exit, disposed object). But I can not find a processor combination that makes it work with the vs2015 apps.
